### DISCUSSION TITLE
.net framework 4.5.2 에서 사용 가능한 가장 최신 버전은?

### DISCUSSION THREAD

**Discussion by baboqoo** [cefsharp/CefSharp#4675] [Q&A]
The most recent version available for use with .NET Framework 4.5.2 that is compatible with Windows 7 and later is version 89.0.17.

**Comment by amaitland** [comment]
> The most recent version available for use with .NET Framework 4.5.2 that is compatible with Windows 7 and later is version 89.0.17.

That's incorrect. `M109` is the last version to support `Windows 7` and is built with `.Net 4.5.2`
